frontend/src/features/browser-profiles/profile-browser.ts

```diff
@@ -147,43 +198,87 @@ export class ProfileBrowser extends LiteElement {
           class="fixed left-1/2 top-2 z-50 flex -translate-x-1/2 items-center rounded-lg bg-white text-base shadow"
         >
           ${this.renderSidebarButton()}
-          <sl-icon-button
-            name="fullscreen-exit"
-            label=${msg("Exit fullscreen")}
-            @click=${() => void document.exitFullscreen()}
-          ></sl-icon-button>
+          <sl-tooltip content=${msg("Exit fullscreen")}>
+            <sl-icon-button
+              name="fullscreen-exit"
+              @click=${() => void document.exitFullscreen()}
+            ></sl-icon-button>
+          </sl-tooltip>
         </div>
       `;
     }
 
     return html`
-      <div class="p-1 text-right text-base">
-        ${this.renderSidebarButton()}
-        <sl-icon-button
-          name="arrows-fullscreen"
-          label=${msg("Enter fullscreen")}
-          @click=${() => void this.enterFullscreen("interactive-browser")}
-        ></sl-icon-button>
+      <div class="flex items-center justify-between">
+        <div class="flex items-center gap-2 px-3 font-medium text-neutral-700">
+          <span id="profileBrowserLabel"> ${msg("Interactive Browser")} </span>
+          <sl-tooltip
+            content=${msg(
+              "Interact with this embedded browser to set up your browser profile. The embedded browser will exit without saving changes after a few minutes of inactivity.",
+            )}
+            hoist
+          >
+            <sl-icon class="text-base" name="info-circle"></sl-icon>
+          </sl-tooltip>
+        </div>
+        <div class="p-1 text-base">
+          ${this.renderSidebarButton()}
+          <sl-tooltip content=${msg("Enter fullscreen")}>
+            <sl-icon-button
+              name="arrows-fullscreen"
+              @click=${() => void this.enterFullscreen()}
+            ></sl-icon-button>
+          </sl-tooltip>
+        </div>
       </div>
     `;
   }
 
   private renderBrowser() {
-    if (this.hasFetchError) {
+    if (this.browserNotAvailable) {
       return html`
-        <btrix-alert variant="danger">
-          ${msg(`The interactive browser is not available.`)}
-        </btrix-alert>
+        <div class="flex h-full w-full items-center justify-center">
+          <btrix-alert variant="danger">
+            <p>
+              ${msg(`Interactive browser is unavailable due to inactivity.`)}
+            </p>
+            <div class="py-2 text-center">
+              <sl-button size="small" @click=${this.onClickReload}>
+                <sl-icon slot="prefix" name="arrow-clockwise"></sl-icon>
```

Comment:
Would prefer to use "re-connect" instead of "reload" as the page isn't being reloaded and reloading can lead to data loss in other apps / contexts.  Here it should continue the session, right?

```suggestion
                ${msg("Reconnect to Browser")}
```

Reply:
No, if the session has ended, it will need to reload the browser, which can lead to data loss, but that's the only thing we can do.